src/overlays/actors/ovl_En_Jc_Mato/z_en_jc_mato.c

```diff
@@ -19,20 +28,111 @@ const ActorInit En_Jc_Mato_InitVars = {
     (ActorFunc)EnJcMato_Init,
     (ActorFunc)EnJcMato_Destroy,
     (ActorFunc)EnJcMato_Update,
-    (ActorFunc)EnJcMato_Draw
+    (ActorFunc)EnJcMato_Draw,
+};
+
+// was D_80B9E210
```

Comment:
Could probably be removed (not sure how others feel)

Reply:
Oops, yeah haha that is a left over comment